app/models/game.rb

```diff
@@ -4,4 +4,38 @@ class Game < ApplicationRecord
   has_one :black_player, class_name: 'User', dependent: false
   has_one :winning_player, class_name: 'User', dependent: false
   has_many :pieces, dependent: false
+
+  attr_accessor :board_positions
+
+  def initialize
+    @board_positions = Array.new(8) { Array.new(8, ' ') }
+  end
+
+  def populate_board!
+    @board_positions[0][0] = Rook.new
```

Comment:
Try something like:

`Rook.create(x_position: 0, y_position: 0, game: self, color: 'black')`

**Things to notice here:**
Am setting `game` to `self`. This is me saying "This piece belongs to this game"
Color - we might not have a color column? I forget. Need to identify the pieces color somehow.

Reply:
@mscoutermarsh `game: self` makes perfect sense - thanks for pointing that out. The color is something Dave and I were discussing the other day, and I don't believe it's something have in the DB yet, but we thought we might add it as part of the "Render the Pieces" Trello card. We'll get on that!